js/modules/k6/grpc/client.go

```diff
@@ -339,13 +347,17 @@ type connectParams struct {
 	IsPlaintext           bool
 	UseReflectionProtocol bool
 	Timeout               time.Duration
+	MaxReceiveSize        int64
```

Comment:
Had to use `int64` instead of `int` since it seems like the string is parsed as `int64` and didn't want to mess with it. If it's not ok please let me know.